---
title: "' Throw ' işleneni ' System. Exception ' öğesinden türetilmelidir"
ms.date: 07/20/2015
f1_keywords:
- vbc30665
- bc30665
helpviewer_keywords:
- BC30665
ms.assetid: 7c228087-39ea-4b30-a410-6ba711e67e5e
ms.openlocfilehash: e9e43605d5b3a752e4024414eeb87dea60d118b3
ms.sourcegitcommit: bf5c5850654187705bc94cc40ebfb62fe346ab02
ms.translationtype: MT
ms.contentlocale: tr-TR
ms.lasthandoff: 09/23/2020
ms.locfileid: "91068914"
---
# <a name="throw-operand-must-derive-from-systemexception"></a>' Throw ' işleneni ' System. Exception ' öğesinden türetilmelidir

İçin sağlanan bağımsız değişken, ' `Throw` ın bir örneği `System.Exception` ya da öğesinden türetilmiş bir sınıfın bir örneği olmalıdır `System.Exception` .  
  
 **Hata kimliği:** BC30665  
  
## <a name="to-correct-this-error"></a>Bu hatayı düzeltmek için  
  
- Aşağıdaki örnekte olduğu gibi, öğesinden türetilmiş bir bağımsız değişken kullanın `System.Exception` .  
  
    ```vb
    Throw New System.Exception("This is an error.")  
    ```  
  
## <a name="see-also"></a>Ayrıca bkz.

- [Throw Deyimi](../language-reference/statements/throw-statement.md)
- [Try...Catch...Finally Deyimi](../language-reference/statements/try-catch-finally-statement.md)
- <xref:System.Exception?displayProperty=nameWithType>
- [.NET 'te özel durumları işleme ve atma](../../standard/exceptions/index.md)
